react/Dialog/DialogEffects.ts

```diff
@@ -45,28 +46,37 @@ const makeOnUnmount = (
   immersive: boolean,
   sidebar: Element | null,
   cozyBar: Element | null
-): FlagshipUI => ({
-  ...(hasAnotherDialog
-    ? {}
-    : {
-        bottomBackground:
-          theme.palette.background[sidebar ? 'default' : 'paper']
-      }),
-  bottomTheme: immersive ? 'light' : 'dark',
-  bottomOverlay: 'transparent',
-  topOverlay: 'transparent',
-  ...(hasAnotherDialog || !cozyBar
-    ? {}
-    : {
-        topBackground:
-          cozyBar &&
-          getComputedStyle(cozyBar).getPropertyValue('background-color')
-      }),
-  topTheme:
-    immersive || (cozyBar && cozyBar.classList.contains('coz-theme-primary'))
+): FlagshipUI => {
+  const otherDialogStyles = getFirstDialogStyles()
+  const isDark =
+    otherDialogStyles !== null &&
+    lightOrDark(otherDialogStyles.color) === 'dark'
```

Comment:
What if otherDialogStyles is null?

Reply:
Then it will default to 'light'. But it's implicit, not clearly visible, I can improve that